python-project-template/pyproject.toml.jinja

```diff
@@ -87,3 +87,6 @@ profile = "black"
 [tool.setuptools.package-data]
 {{package_name}} = ["py.typed"]
 {%- endif %}
+
+[tool.coverage.run]
+omit=["src/{{project_name}}/_version.py"]
```

Comment:
I think this might need to be the package name. Similar to line 69 in this file. 
```suggestion
omit=["src/{{package_name}}/_version.py"]
```